src/screens/Voucher/VoucherDetailScreen.tsx

```diff
@@ -43,7 +45,12 @@ export const VoucherDetailScreen = ({ route }: StackScreenProps<any>) => {
     return <LoadingSpinner loading />;
   }
 
-  const { contentBlocks, discountType, quota, id } = data[QUERY_TYPES.GENERIC_ITEM];
+  const { contentBlocks, discountType, quota, id } = data[QUERY_TYPES.GENERIC_ITEM] || data;
```

Comment:
is `|| data` really needed here?

Reply:
Yes. You're right, this control can be removed.